packages/manager/modules/octavia-load-balancer/src/components/instances-table/instances-table.html

```diff
@@ -7,137 +7,139 @@
-                            >
-                            </oui-select>
-                        </div>
-                        <div class="col-3">
-                            <label
-                                for="select-listener-protocol"
-                                class="oui-label d-flex align-items-center mb-1"
-                            >
-                                <small
-                                    data-translate="octavia_load_balancer_instances_table_port_label"
-                                    class="mr-1"
-                                ></small>
-                                <span
-                                    class="oui-icon oui-icon-help"
-                                    oui-tooltip="{{:: 'octavia_load_balancer_instances_table_port_tooltip' | translate }}"
-                                ></span>
-                            </label>
-                            <input
-                                class="oui-input"
-                                type="text"
-                                name="select-listener-port"
-                                data-ng-model="$ctrl.listeners[$index].port"
-                            />
-                        </div>
-                        <div class="col-3 align-self-end">
-                            <oui-button
-                                variant="link"
-                                data-aria-label="{{:: 'octavia_load_balancer_instances_table_listener_delete' | translate}}"
-                                data-on-click="$ctrl.deleteListener($index)"
-                                oui-tooltip="{{:: 'octavia_load_balancer_instances_table_listener_delete' | translate }}"
-                            >
-                                <span
-                                    class="oui-icon oui-icon-bin"
-                                    aria-hidden
-                                ></span>
-                            </oui-button>
+<div class="container-fluid">
+    <table class="oui-table">
+        <thead>
+            <tr>
+                <th class="oui-table__header">
+                    <span
+                        data-translate="octavia_load_balancer_instances_table_listener"
+                    ></span>
+                </th>
+                <th class="oui-table__header"></th>
+                <th class="oui-table__header">
+                    <span
+                        data-translate="octavia_load_balancer_instances_table_pools"
+                    ></span>
+                    <span
+                        class="oui-icon oui-icon-help"
+                        oui-tooltip="{{:: 'octavia_load_balancer_instances_table_pools_tooltip' | translate }}"
+                    ></span>
+                </th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr
+                data-ng-repeat="listener in $ctrl.listeners track by $index"
```

Comment:
No side effect if the array is not always sort in the same way ?

Reply:
nop the different lines have no dependencies with each others